Plots/XY/NCL_tm_1.py

```diff
@@ -0,0 +1,128 @@
+###############################################################################
+# Plot 1
+
+# Generate figure (set its size (width, height) in inches) and axes
+plt.figure(1, figsize=(8, 3.5))
+ax = plt.gca()
+
+# Format the tick labels. Use integers for the major ticks.
+# For the minor ticks, use no labels; defaults to NullFormatter.
+ax.xaxis.set_major_formatter(FormatStrFormatter('%d'))
+ax.yaxis.set_major_formatter(FormatStrFormatter('%.1f'))
+
+# Set the major tick spacing.
+major_tick_spacing = 10
+ax.xaxis.set_major_locator(MultipleLocator(major_tick_spacing))
+
+# Draw ticks on all sides of the plot.
+plt.tick_params(which='both', top=True, right=True)
+
+# Increase the length of the tick marks.
+plt.tick_params(which='major', length=10.0, width=0.5)
+plt.tick_params(which='minor', length=5.0, width=0.25)
+
+# Set label size
+plt.xticks(fontsize=14)
+plt.yticks(fontsize=14)
+
+# Set the minor tick spacing for X and Y axes.
+ax.xaxis.set_minor_locator(MultipleLocator(2))
+ax.yaxis.set_minor_locator(MultipleLocator(0.5))
+
+# Plot data and set the X axis limits.
+plt.plot(x_data, y_data, color='grey', linewidth=0.5)
+
+# Usa geocat.viz.util convenience function to set axes parameters without calling several matplotlib functions
+# Set axes limits
+gvutil.set_axes_limits_and_ticks(ax,
+                                 xlim=(min(x_data) - 1, max(x_data) + 1),
+                                 ylim=(-4.5, 4.5))
+
+# Draw plot on the screen
+plt.tight_layout()
+plt.show()
+
+###############################################################################
+# Plot 2 (forced tickmark label at the beginning of X axis)
+
+# Generate figure (set its size (width, height) in inches) and axes
+plt.figure(2, figsize=(8, 3.5))
+ax = plt.gca()
+
+# Format the tick labels.
+# For the minor ticks, use no labels; defaults to NullFormatter.
+ax.xaxis.set_major_formatter(FormatStrFormatter('%d'))
+ax.yaxis.set_major_formatter(FormatStrFormatter('%.1f'))
+
+# Set the major tick spacing.
+major_tick_spacing = 10
+xticks = [1949, 1959, 1969, 1979, 1989, 1999]
+ax.xaxis.set_major_locator(FixedLocator(xticks))
+
+# Draw ticks on all sides of the plot.
+plt.tick_params(which='both', top=True, right=True)
+
+# Increase the length of the tick marks.
+plt.tick_params(which='major', length=10.0, width=0.5)
+plt.tick_params(which='minor', length=5.0, width=0.25)
+
+# Set label size
+plt.xticks(fontsize=14)
+plt.yticks(fontsize=14)
+
+# Set the minor tick spacing for X and Y axes.
+ax.xaxis.set_minor_locator(AutoMinorLocator(4))
+ax.yaxis.set_minor_locator(MultipleLocator(0.5))
+
+# Line-plot data
+plt.plot(x_data, y_data, color='grey', linewidth=0.5)
+
+# Use geocat.viz.util convenience function to set axes parameters without calling several matplotlib functions
+# Set axes limits, and tick values on x-axes.
+gvutil.set_axes_limits_and_ticks(ax,
+                                 xlim=(min(x_data) - 1, max(x_data) + 1),
+                                 ylim=(-4.5, 4.5))
```

Comment:
```suggestion
                                 ylim=(-4.5, 4.5)
                                 yticks= [-4, -2, 0, 2, 4],
                                 yticklabels=['-4.0', '-2.0', '0.0',  '2.0', '4.0'])
```
Define the correct y tick labels to match NCL plot. This should also be implemented in the first plot.